tests/test_core.py

```diff
@@ -8181,7 +8181,7 @@ def test_safe_stack_dylink(self):
         int a[2048];
         f(a);
       }
-    ''', ['abort(stack overflow)', '__handle_stack_overflow'], assert_returncode=NON_ZERO, force_c=True)
+    ''', ['abort(stack overflow)', '__handle_stack_overflow'], assert_returncode=NON_ZERO, force_c=True, need_reverse=False)
```

Comment:
Is this addition of `need_reverse=False` related to this PR?

Reply:
Ooops, no idea how this stuff snuck in.   Will revert.

Reply:
Thanks for the keen eye.. i completely missed that I had mixed up other changes in this PR!